### PR TITLE
Search screen and Navigation Bar

### PR DESCRIPTION
**Pull request by aryanagarwal15**
Added color to AppBar and Drawer
Search bar changed when search button is clicked

### CODE REVIEW

File: swd/frontend/src/Components/Navigation/Navigation.js [hunk 1]
```diff
@@ -79,29 +79,37 @@ constructor(props) {
       width: 24,
       height: 24,
     };
-    
+      const toolbarStyle = {
+          backgroundColor: "#00BCD4"
```
Comment (prampey) [inline swd/frontend/src/Components/Navigation/Navigation.js]: Colour can be imported from material-ui as cyan500. Check Figma for the colour palette.

File: swd/frontend/src/Components/Navigation/Navigation.js [hunk 2]
```diff
@@ -79,29 +79,37 @@ constructor(props) {
...
+          <Toolbar style={toolbarStyle}>
             <ToolbarGroup firstChild>
-              <IconButton onClick={this.handleSideBarToggle}><NavigationMenu color={darkGreen} /></IconButton>
+                {!this.state.searchMode?
+                    <IconButton onClick={this.handleSideBarToggle}><NavigationMenu color={"#fff"} /></IconButton>
+                    :<IconButton onClick={this.handleCloseSearch} ><img src={backIcon} height={25}/></IconButton>}
+
               <Sidebar open={this.state.sidebarOpen} toggleOpen={this.handleSideBarToggle} />
-              {this.state.searchMode?<SearchBarWithAnimation style={{width: '90%'}}/>:<ToolbarTitle text="SWD" />}
+              {this.state.searchMode?<SearchBarWithAnimation style={{width: '80%'}}/>:<ToolbarTitle style={appBarColor} text="Student Welfare Division" />}
```
Comment (prampey) [inline swd/frontend/src/Components/Navigation/Navigation.js]: Toolbar is not responsive. So you have 2 ways to fix this:
- Change "Student Welfare Division" title to "SWD" and there will be no occlusion on the right of the toolbar or,
- Try to find a way to make Toolbar responsive.

File: swd/frontend/src/Components/Navigation/Navigation.js [hunk 3]
```diff
@@ -79,29 +79,37 @@ constructor(props) {
...
               {this.state.searchMode?<span/>:<ToolbarSeparator style={{marginLeft:0}} />}
             </ToolbarGroup>
             <ToolbarGroup lastChild style={{position: 'relative'}}>
-              {this.state.searchMode?<span/>:<Link to='/search/' style={{position: 'absolute', left: 5}}><IconButton iconStyle={filledIcon} style={{paddingLeft: 0, paddingRight: 20}} onClick={this.handleSearch}><ActionSearch color={darkGreen} /></IconButton></Link>}
+              {this.state.searchMode?<span/>:<Link to='/search/' style={{position: 'absolute', left: 5}}><IconButton iconStyle={filledIcon} style={{paddingLeft: 0, paddingRight: 20}} onClick={this.handleSearch}><ActionSearch color={"#fff"} /></IconButton></Link>}
```
Comment (prampey) [inline swd/frontend/src/Components/Navigation/Navigation.js]: Please check out the positioning of the search icon. It seems to be pushed above the toolbar. 
Also there seem to be too many dividers.

**Comment by prampey** [comment]
This is the current state. Search icon needs to be repositioned and AppBar needs to be made responsive.
screen shot 2018-03-21 at 12 42 59

**Comment by aryanagarwal15** [comment]
@prampey I have made requested changes. Please review. In my instance the position of the search button is absolutely fine. Inform if the issue still exists.

screenshot from 2018-03-21 13-45-34

**Comment by prampey** [comment]
Looks I get it only on Firefox mobile mode. I'll open up an issue. We'll look into it later.

**Pull request merged**